actors/builtin/miner/policy.go

```diff
@@ -81,47 +91,67 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
 	abi.RegisteredSealProof_StackedDrg64GiBV1:  abi.ChainEpoch(10000),
 }
 
-// Number of epochs between publishing the precommit and when the challenge for interactive PoRep is drawn
-// used to ensure it is not predictable by miner.
+// PARAM_SPEC
+// Number of epochs between publishing the precommit and when the challenge for interactive PoRep is drawn.
+// Motivation: This guarantees that (1) a miner cannot predict a future challenge (2) a miner cannot do a long fork in the past to insert a precommit after seeing a challenge.
 const PreCommitChallengeDelay = abi.ChainEpoch(150)
 
 // Lookback from the current epoch for state view for leader elections.
 const ElectionLookback = abi.ChainEpoch(1) // PARAM_FINISH
 
-// Lookback from the deadline's challenge window opening from which to sample chain randomness for the challenge seed.
+// PARAM_SPEC
+// Lookback from the deadline's challenge window opening from which to sample chain randomness for the WindowPoSt challenge seed.
 // This lookback exists so that deadline windows can be non-overlapping (which make the programming simpler)
-// but without making the miner wait for chain stability before being able to start on PoSt computation.
-// The challenge is available this many epochs before the window is actually open to receiving a PoSt.
+// Motivation: A miner can start the WindowPoSt computation without waiting for chain stability. This value cannot be too large since it could compromise the rationality of honest storage (due to WindowPoSt cost assumptions)
 const WPoStChallengeLookback = abi.ChainEpoch(20)
```

Comment:
**Confirmation needed:** @irenegia to confirm that 20 + 60 epochs (assuming 30s block time) is still safe in the cost model for WindowPoSt (if I recall correctly we allocated 1h and 60+20 is about 40 mins)

Reply:
Confirming that having ~40 mins to submit the proof (i.e., 40 mins between the time the challenges are received and the deadline) is fine.